src/pages/public_registration/preview_file/PreviewOfficeFile.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styled from 'styled-components';
+import { CommonPreviewProps } from './PreviewFile';
+
+const StyledIframe = styled.iframe`
+  width: 100%;
+  height: 25rem;
+`;
```

Comment:
😫 hvor godt er denne testet? Mener iPhone ikke støtter iframes

Reply:
Bør kanskje ha en sånn «kan ikke forhåndsvise fil» her også isåfall?

Reply:
hmmm, spennende... kan du prøve denne på iphone? https://view.officeapps.live.com/op/embed.aspx?src=https://calibre-ebook.com/downloads/demos/demo.docx
(det er en demo, og akkurat samme som vi bruker, bare full size)

Reply:
Den funker ja. Men tror det er selve iframen som blir blokkert av Safari. Kan se om jeg finner en iframe-demo

Reply:
Fila vises i en iframe i linken der, de ser det hvis du inspiserer element

Reply:
Åja, ok, men da er det kanskje fikset da 👍🏼